.github/workflows/check_translation_format.yml

```diff
@@ -0,0 +1,42 @@
+name: Check format
```

Comment:
Let's keep this file's name as `frappe_linter` so that we can add more similar linters?